src/packs/rewards/Treasures_R5dDUb9qLRmp5fQo/Leveled_Treasures_GR0S5ItmIn79Gxk1/Weapons_78VTHX5tnOWffNOa/treasure_Authority_s_End_UIfbglFC3H7BmDfY.json

```
@@ -3,11 +3,259 @@
+      "type": "abilityModifier",
+      "origin": "Compendium.draw-steel.rewards.Item.UIfbglFC3H7BmDfY",
+      "_id": "wkRs1sOiZ8IXhnAs",
+      "system": {
+        "end": {
+          "type": "",
+          "roll": "1d10 + @combat.save.bonus"
+        },
+        "filters": {
+          "keywords": [
+            "weapon"
+          ]
+        }
+      },
+      "changes": [
+        {
+          "key": "damage.tier1.value",
+          "mode": 2,
+          "value": "3",
+          "priority": null
+        },
+        {
+          "key": "damage.tier2.value",
+          "mode": 2,
+          "value": "3",
+          "priority": null
+        },
+        {
+          "key": "damage.tier3.value",
+          "mode": 2,
+          "value": "3",
+          "priority": null
+        }
+      ],
+      "disabled": true,
+      "duration": {
+        "startTime": null,
+        "combat": null,
+        "seconds": null,
+        "rounds": null,
+        "turns": null,
+        "startRound": null,
+        "startTurn": null
+      },
+      "description": "",
+      "tint": "#ffffff",
+      "transfer": true,
+      "statuses": [],
+      "sort": 0,
+      "flags": {},
+      "_stats": {
+        "compendiumSource": null,
+        "duplicateSource": null,
+        "exportSource": null,
+        "coreVersion": "13.351",
+        "systemId": "draw-steel",
+        "systemVersion": "0.10.0",
+        "createdTime": 1764262618181,
+        "modifiedTime": null,
+        "lastModifiedBy": null
+      },
+      "_key": "!items.effects!UIfbglFC3H7BmDfY.wkRs1sOiZ8IXhnAs"
+    },
+    {
+      "name": "Authority’s End",
+      "img": "icons/weapons/misc/whip-red-yellow.webp",
+      "type": "base",
+      "origin": "Compendium.draw-steel.rewards.Item.UIfbglFC3H7BmDfY",
+      "transfer": false,
+      "_id": "om8bpobSQ2agKz2C",
+      "system": {
+        "end": {
+          "type": "encounter",
+          "roll": "1d10 + @combat.save.bonus"
+        }
+      },
+      "changes": [
+        {
+          "key": "system.combat.save.bonus",
+          "mode": 2,
+          "value": "1",
+          "priority": null
+        }
+      ],
+      "disabled": false,
+      "duration": {
+        "startTime": null,
+        "combat": null,
+        "seconds": null,
+        "rounds": null,
+        "turns": null,
+        "startRound": null,
+        "startTurn": null
+      },
+      "description": "<p><span style=\"color:rgb(239, 230, 216);font-family:Signika, 'Palatino Linotype', sans-serif;font-size:14px;font-style:normal;font-variant-ligatures:normal;font-variant-caps:normal;font-weight:400;letter-spacing:normal;orphans:2;text-align:start;text-indent:0px;text-transform:none;widows:2;word-spacing:0px;-webkit-text-stroke-width:0px;white-space:normal;background-color:rgba(11, 10, 19, 0.9);text-decoration-thickness:initial;text-decoration-style:initial;text-decoration-color:initial;display:inline !important;float:none\">Gains a +1 bonus to saving throws.</span></p>",
```

```suggestion
      "description": "<p>Gains a +1 bonus to saving throws.</p>",
```
Need to use ctrl-shift-v or the "remove styling" button